adapters/ix/ix.go

```diff
@@ -359,3 +508,104 @@ func BuildIxDiag(request *openrtb2.BidRequest) error {
 	}
 	return nil
 }
+
+// moves sid from imp[].ext.bidder.sid to imp[].ext.sid
+func moveSid(imp *openrtb2.Imp) error {
+	if imp.Ext == nil || len(imp.Ext) == 0 {
```

Comment:
Nitpick: This can just be `if len(imp.Ext) == 0 {` since `len` gracefully handles the `nil` case.